docs/user/alerting/troubleshooting/alerting-common-issues.asciidoc

```diff
@@ -266,3 +266,23 @@ This error happens when the `xpack.encryptedSavedObjects.encryptionKey` value us
 | Generate a new API key for the rule by disabling then enabling the rule.
 
 |===
+
+[float]
+[[known-issue-upgrade-rule]]
+==== Rules stop running after upgrade
+
+*Problem*:
+
+Alerting rules that were created or edited in 8.2 stop running after you upgrade
+to later releases. The following error occurs:
```

Comment:
Should this be more specific? `later releases` makes it seem like all later releases when it is `8.3.0` and `8.3.1` that are the problem releases.

Reply:
I think you're right that it's only incurred by upgrading to 8.3.0 or 8.3.1 so I've created https://github.com/elastic/kibana/pull/140674